web/themes/custom/hoeringsportal/templates/block/block-content--aside-contact-info.html.twig

```diff
@@ -28,32 +28,36 @@
  * @ingroup themeable
  */
 #}
-
-<div class="aside-contact-info">
-  {% if block_content.field_department.value %}
-    <div class="text-primary">
-      {{ block_content.field_department.value }}
-    </div>
-  {% endif %}
-  {% if block_content.field_title.value %}
-    <div class="h4">
-      {{ block_content.field_title.value }}
-    </div>
-  {% endif %}
-  {% if block_content.field_address.value %}
-    <div>
-      {# @TODO Which is better; `nl2br` or `style="white-space: pre-line"` #}
-      {{ block_content.field_address.value|nl2br }}
-    </div>
-  {% endif %}
-  {% if block_content.field_phone_number.value %}
-    <div>
-      {{ 'Phone'|trans }}: <a href="tel:{{ block_content.field_phone_number.value }}">{{ block_content.field_phone_number.value }}</a>
-    </div>
-  {% endif %}
-  {% if block_content.field_email.value %}
-    <div>
-      {{ 'Email'|trans }}: <a href="mailto:{{ block_content.field_email.value }}">{{ block_content.field_email.value }}</a>
-    </div>
+<div class="aside-contact-info d-flex flex-column">
+  {% if block_content.field_image %}
```

Comment:
Are you sure that `block_content.field_image` is not set if no image has been selected? Drupal is weird at times …

Reply:
Good catch - It was still set. Changed it to check the file_url